### PR TITLE
How to integrate a new cluster in an existing VNET

### PR DESCRIPTION
**Pull request by jcorioland**
add documentation to explain how to use acs engine to integrate a cluster into an existing virtual network

---
This change is [Reviewable](https://reviewable.io/reviews/azure/acs-engine/506)

### CODE REVIEW

**Comment by jcorioland** [comment]
Hey @squillace, the PR for custom vnet is ready on my side :) tell me if I need to update something. 
Thanks

**Comment by JackQuincy** [comment]
@colemickens @anhowe Can one of you take a look at the documentation? I'm not an expert here.

File: docs/custom-vnet.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
+# Using a custom virtual network with Azure Container Service 
+In this tutorial you are going to learn how to use [ACS Engine](https://github.com/Azure/acs-engine) to deploy a brand new cluster into an existing or pre-created virtual network. 
+By doing this, you will be able to control the properties of the virtual network or integrate a new cluster into your existing infrastructure.
+
+*Note: This article describes the procedure with Docker Swarm but it will work in the exact same way with the all the orchestrators available with ACS Engine: Docker Swarm, Kubernetes and DC/OS.*
```
Comment (anhowe) [inline docs/custom-vnet.md]: Put a note here that for Kubernetes, it must be deployed in same RG, and permissions must be set according to the kubernetes walkthrough.

**Comment by jcorioland** [comment]
@anhowe, I have added the link :)

File: docs/custom-vnet.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Swarm"
+    },
+    "masterProfile": {
+      "count": 3,
+      "dnsPrefix": "",
+      "vmSize": "Standard_D2_v2",
+      "vnetSubnetId": "/subscriptions/SUBSCRIPTION_ID/resourceGroups/RESOURCE_GROUP_NAME/providers/Microsoft.Network/virtualNetworks/ExampleCustomVNET/subnets/ExampleMasterSubnet",
```
Comment (rjtsdl) [inline docs/custom-vnet.md]: is it possible to make this line highlighted?

Comment (shrutir25) [inline docs/custom-vnet.md]: @rjtsdl - could you look into this ?

Comment (shrutir25) [inline docs/custom-vnet.md]: @rjtsdl -how do we want to proceed with this PR ?

Comment (rjtsdl) [inline docs/custom-vnet.md]: NVM, i don't have a way to highlight as well.

**Comment by anhowe** [comment]
@acs-bot test me please

**Comment by shrutir25** [comment]
@acs-bot test me please

**Comment by jchauncey** [comment]
Not real sure how to kick this off in circle to get the status to update

@seanknox  thoughts?

**Comment by seanknox** [comment]
Closed and reopened, looks to be running.

**Pull request merged**